test/functional/feature_csv_activation.py

```diff
@@ -9,8 +9,8 @@
 BIP 112 - CHECKSEQUENCEVERIFY
 BIP 113 - MedianTimePast semantics for nLockTime
 
-mine 82 blocks whose coinbases will be used to generate inputs for our tests
-mine 345 blocks and seed block chain with the 82 inputs will use for our tests at height 427
+mine 83 blocks whose coinbases will be used to generate inputs for our tests
```

Comment:
just a note:
```
        consensus.CSVHeight = 432; // CSV activated on regtest (Used in rpc activation tests)
```
same for `dash` and for `bitcoin`, so, comment is valid and requires no dashification.